universal-application-tool-0.0.1/project/Tailwindbuilder.scala

```diff
@@ -3,20 +3,14 @@ import sbt.File
 
 import scala.sys.process.Process
 
-object Webpack {
+object Tailwindbuilder {
```

Comment:
suggestion (blocking): Use camel case here: `TailwindBuilder`. `TailwindCssBuilder` might be even better.

Reply:
Appreciate the comments, I will make changes accordingly.